DeepSpeech.ipynb

```diff
@@ -1274,10 +1255,11 @@
     "def run_inference(session, caption, data_set, results_params=None):\n",
     "    if results_params is None:\n",
     "        # Get the data_set specific graph end-points\n",
-    "        results_params = get_results_params(data_set)\n",
+    "        tower_decodings, tower_labels, _, tower_distances, tower_total_losses, _, _ = get_tower_results(data_set)\n",
```

Comment:
This hasn't changed in this patch-set, but I worry about recreating this graph every time this function is called, rather than just doing it once. I suppose the flip-side is it can get garbage-collected and associated resources would be freed and the cost of creating the graph over the entire run is miniscule... I have no strong feelings here, as long as you've considered it.